state_machine/applications/flight/Tasks/beacon_task.py

```diff
@@ -27,8 +28,8 @@ class task(Task):
     def __init__(self, satellite):
         super().__init__(satellite)
         # set our radiohead node ID so we can get ACKs
-        self.cubesat.radio.node = 0xFA  # our ID
-        self.cubesat.radio.destination = 0xAB  # target's ID
+        cubesat.radio.node = 0xFA  # our ID
```

Comment:
I don't think we need wrapper functions, I'd really like to avoid adding stuff like that especially since we are memory constrained. 
When we do `send = _cubesat.radio.send` we're using up just one pointers worth of memory. 
But when we do a whole wrapper function I suspect we are doing much much more (unless python is very clever)

Reply:
I agree. I think I mentioned this in a later comment but it feels redundant to create all these functions and I think just creating a radio accessor (radio = _cubesat.radio) is much more elegant.

Reply:
I don't really feel strongly in either direction.